copilot/fastapi/run_tests.sh

```
@@ -0,0 +1,85 @@
+RED='\033[0;31m'
+YELLOW='\033[1;33m'
+NC='\033[0m' # No Color
+
+echo -e "${YELLOW}Setting up test environment...${NC}"
+
+# Set working directory to the script location
+cd "$(dirname "$0")"
+
+# Make sure we're using the correct Python path
+# Update to include the app directory explicitly in addition to parent and current dir
+export PYTHONPATH=$(cd .. && pwd):$(pwd):$(pwd)/app
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)